platforms/intel-legacy/broadwell/index.rst

```diff
@@ -8,6 +8,6 @@ EP DSP integrated in the PCH. It relies only on internal memory (no
 caches).
 
 Supported commercially-available platforms are limited. The SOF team
-is only aware of the Chromebook Pixel 2015 (code name "samus") and the
-Dell XPS.
+is aware of the Chromebook Pixel 2015 (code name "samus") and a few
+tablets and convertibles from Dell and HP.
```

Comment:
such as


    Dell Latitude 7350 - convertible
    Dell Venue 11 Pro 7140 - tablet
    HP Spectre x360 (2015) - convertible

?

Reply:
Sure, I changed it to list all known devices (and I used the manufacturer's name for each).